netdata/plan.sh

```diff
@@ -0,0 +1,69 @@
+pkg_name=netdata
+pkg_origin=core
+pkg_version="1.10.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-3.0")
```

Comment:
Could you update this to `GPL-3.0-or-later` to align with https://spdx.org/licenses/

Reply:
@themightychris gentle ping on this 😸

Reply:
@smacfarlane done